rllib/env/multi_agent_env.py

```diff
@@ -434,7 +434,13 @@ def make_multi_agent(
     class MultiEnv(MultiAgentEnv):
         def __init__(self, config=None):
```

Comment:
What is the type of config here? is it not a normal dict? in any case please add the type via config: Type

Reply:
it's EnvContext